src/lib/components/Text.svelte

```diff
@@ -8,15 +8,29 @@
         logo: [
           'font-inter', 'font-semibold', 'text-5xl', 'sm:text-6xl', 'tracking-logo',
         ],
-        heading: [
-          'font-geist', 'font-medium', 'text-4xl', 'sm:text-5xl', 
+        hero: [
+        'font-geist', 'font-medium', 'text-3xl', 'sm:text-7xl',
+        ],
+        title1: [
```

Comment:
Why not just use `title` for now since we don't have any other styles for other titles? 🤔 

Or we can even use `heading` and `subheading` (instead of `subhead`), not sure if using numbers in these is automatically clear to me, but that might be because I haven't spend a lot of time in Frontend-land lately.

Reply:
it has to do with the [typography styles ](https://www.figma.com/design/4mgL0nrZAdWpQyzxv4l8Lk/Awesomic_OnePlusOne_DashBoard_Design-System?node-id=2087-557&t=DvPvDgWyRuUpg78w-4)Mattheus is working on